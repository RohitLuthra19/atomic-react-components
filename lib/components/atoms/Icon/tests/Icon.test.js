import React from 'react';
import { shallow } from 'enzyme';
import { ReactSVG } from 'react-svg';
// import { IconVanilla } from '../index';

describe('<Icon />', () => {
  test('should render correctly', () => {
    const IconComponentWrapper = shallow(<ReactSVG title="play">Test</ReactSVG>);
    expect(IconComponentWrapper).toMatchSnapshot();
  });

  it('should have id', () => {
    const IconComponentWrapper = shallow(<ReactSVG id="pause">Test</ReactSVG>);
    expect(IconComponentWrapper.props()).toHaveProperty('id');
  });
});
