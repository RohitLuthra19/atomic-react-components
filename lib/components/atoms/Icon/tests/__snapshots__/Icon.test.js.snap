// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Icon /> should render correctly 1`] = `
<div
  title="play"
/>
`;
